tests/contrib/funsor/test_vectorized_markov.py

```diff
@@ -327,6 +332,14 @@ def test_vectorized_markov(model, data, var, history, use_replay):
             expected_step |= frozenset({v_step})
         assert actual_step == expected_step
 
+    with pyro_backend("contrib.funsor"):
+        if model != model_5:
```

Comment:
nit: instead of putting everything here inside this `if` statement, add a standalone `if` that triggers an `xfail`, e.g.
```py
with pyro_backend("contrib.funsor"):
    if history > 1:
        pytest.xfail(reason="TraceMarkovEnum_ELBO does not yet support history > 1")

    elbo = ...
```